app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -126,13 +137,93 @@ class PatientSummaryScreenControllerTest {
     uiEvents.onNext(PatientSummaryScreenCreated(patientUuid, caller = PatientSummaryCaller.NEW_PATIENT))
 
     verify(screen).populateList(
+        any(),
         any(),
         check {
           it.forEachIndexed { i, item -> assertThat(item.measurement).isEqualTo(bloodPressureMeasurements[i]) }
         },
         any())
   }
 
+  @Test
+  @Parameters(method = "params for placeholder bp items")
+  fun `the placeholder blood pressure items must be shown`(
+      bloodPressureMeasurements: List<BloodPressureMeasurement>,
+      expectedPlaceholderItems: List<SummaryBloodPressurePlaceholderListItem>,
+      expectedBloodPressureMeasurementItems: List<SummaryBloodPressureListItem>
+  ) {
+    whenever(bpRepository.newest100MeasurementsForPatient(patientUuid)).thenReturn(Observable.just(bloodPressureMeasurements))
+    whenever(prescriptionRepository.newestPrescriptionsForPatient(patientUuid)).thenReturn(Observable.just(emptyList()))
+    whenever(medicalHistoryRepository.historyForPatientOrDefault(patientUuid)).thenReturn(Observable.just(PatientMocker.medicalHistory()))
+
+    uiEvents.onNext(PatientSummaryScreenCreated(patientUuid, caller = PatientSummaryCaller.NEW_PATIENT))
+
+    verify(screen).populateList(
+        prescribedDrugsItem = any(),
+        measurementPlaceholderItems = eq(expectedPlaceholderItems),
+        measurementItems = check {
+          it.forEachIndexed { index, item -> assertThat(item.measurement).isEqualTo(expectedBloodPressureMeasurementItems[index].measurement) }
+        },
+        medicalHistoryItem = any()
+    )
+  }
+
+  @Suppress("Unused")
+  private fun `params for placeholder bp items`(): List<List<Any>> {
+    val bpsForTest2 = listOf(PatientMocker.bp(patientUuid))
```

Comment:
Is `bpsForTest1` also present somewhere?

Reply:
Well, it's meant to be `bps for test #2`. `bps for test 1` is an empty list. This will anyway be removed when I change it to dynamically generated tests.